analyses/fusion_filtering/04-project-specific-filtering.Rmd

```diff
@@ -132,7 +137,13 @@ putative_driver_annotated_fusions <- fusion_calls %>%
   dplyr::filter(!is.na(Gene1A_anno) | !is.na(Gene1B_anno) | !is.na(Gene2A_anno) | !is.na(Gene2B_anno)) %>%
   unique()
 
+# filter fusions found in more than 4 histologies
+FusionInNhist<-putative_driver_annotated_fusions %>% dplyr::select(FusionName,broad_histology) %>% unique() %>% group_by(FusionName) %>% summarise(count=n())
```

Comment:
```suggestion
FusionInNhist<-putative_driver_annotated_fusions %>% dplyr::select(FusionName,group) %>% unique() %>% group_by(FusionName) %>% summarise(count=n())
```
Since variable `group` has been was declared and assigned value with `broad_histology` at 
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/53c99c2deab4864706cf426fb9fea2e966fe5bf0/analyses/fusion_filtering/04-project-specific-filtering.Rmd#L9-L12 and https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/53c99c2deab4864706cf426fb9fea2e966fe5bf0/analyses/fusion_filtering/04-project-specific-filtering.Rmd#L89

We should use the signed variable to make the script more generic. 

suggested make similar changes for 
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/53c99c2deab4864706cf426fb9fea2e966fe5bf0/analyses/fusion_filtering/04-project-specific-filtering.Rmd#L93
and 
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/53c99c2deab4864706cf426fb9fea2e966fe5bf0/analyses/fusion_filtering/04-project-specific-filtering.Rmd#L262

Reply:
Ah yes! I'll make these changes